### ISSUE TITLE
some questions in decoder_ldap_der.py

### ISSUE THREAD

**Issue by dreamerlucy** [richm/scripts#25]

**Comment by dreamerlucy** [comment]
when I run decoder_ldap_der.py, it always shows 'you need the pyasn1 package'.
I try several times and find the reason is 'import ldap' . Only pip install python-ldap, but it will show the error:
'this not the right ldap.py module from pyasn1
e.g. set PYTHONPATH to include /usr/share/doc/python-pyasn1-0.0.12a' 
I also set PYTHONPATH, but it doesn't work. 
I want to know how to run this script.
If you could answer my questions, I will be grateful.
Thank you.

**Comment by richm** [comment]
I haven't used decoder_ldap_der.py - looks like there is some code rot - not sure how to make it work with more recent python - I don't use it anymore

**Comment by dreamerlucy** [comment]
thank you for your response.
Then you know what other scripts that can be used to parse the ldap protocol in pcap?

**Comment by richm** [comment]
No, I don't, but you might be able to hack decoder_ldap_der.py to make it work, and if you do, please submit a PR.
